components/Tabs/tests/unit/MDCTabBarTests.m

```diff
@@ -71,4 +73,16 @@ - (void)testTraitCollectionDidChangeBlockCalledWithExpectedParameters {
   XCTAssertEqual(passedTraitCollection, fakeTraitCollection);
 }
 
+- (void)testSettingBaseOverrideBaseElevationReturnsSetValue {
```

Comment:
Tip: @yarneo I assume if a component can never change elevation, then its block can never be called?